html-block-and-inline/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Block and Inline</title>
+</head>
+  <body>
+    <header></header>
+    <h1>Blue Americana Snuggie&reg; Blanket</h1>
+    <h2>by SnuggieStore</h2>
+    <span>Price:<del>$19.99</del> $14.99</span>
```

Comment:
The only thing I would change is to make this `span` a `div` The reason is you want a block level element to contain the inline elements.